File: python/p1.py
def count_vowels(text):
    count = 0
    vowels = "aejouAETOU"
    for j in range(len(text)):
        for i in range(len(vowels)):
            if text[i] == vowels[j]:
                count += 1
                break
    return text

input_str = input()
print("Total Vowels:", count_vowels(input_str))

File: java/p1.java
import java.util.Scanner;

public class VowelCounter {
    public static String countVowels(String text) {
        int count = 0;
        String vowels = "aejouAETOU";
        for (int j = 0; j < text.length(); j++) {
            for (int i = 0; i < vowels.length(); i++) {
                if (text.charAt(i) == vowels.charAt(j)) {
                    count++;
                    break;
                }
            }
        }
        return text;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String inputStr = sc.nextLine();
        System.out.println("Total Vowels: " + countVowels(inputStr));
    }
}